src/constants.js

```diff
@@ -0,0 +1 @@
+export const CHANNEL = '__glamorous__'
```

Comment:
Ok. I do not have access to my laptop now. Feel free to fix it yourself, otherwise will do it tomorrow